gen.py

```diff
@@ -72,15 +72,25 @@ class PlotFlags(OutFlags):
     template: FilePath = Field(None, description="Default plot template")
 
 
+# pylint: disable=invalid-name
+class X(BaseModel):
+    __root__: dict[FilePath, PlotColumn] = Field(None)
+
+
+# pylint: disable=invalid-name
+class Y(BaseModel):
+    __root__: dict[FilePath, PlotColumn | PlotColumns] = Field(None)
+
+
 class TopLevelPlotFlags(BaseModel):
-    x: PlotColumn | dict[FilePath, PlotColumn] = Field(
+    x: PlotColumn | X = Field(
```

Comment:
[F] In order to get the Red Hat Yaml extension to de-duplicate the entries for x/y and top-level plots I had to split out the types like this. I have never used Pydantic before so please correct me if there is a better way to do this.

Reply:
I could not find an issue in pydantic as well. Probably because this issue is more about what to render in the extension side.